rustup-init.sh

```diff
@@ -26,47 +26,36 @@ RUSTUP_UPDATE_ROOT="${RUSTUP_UPDATE_ROOT:-https://static.rust-lang.org/rustup}"
 usage() {
     cat <<EOF
 rustup-init 1.26.0 (577bf51ae 2023-04-05)
```

Comment:
> The only caveat that isn't looking ok is the removal of the rustup version details from the top of the help. Its not necessarily a show stopper, but it does make me nervous, sometimes we have behavioural expectations in the oddest places.
>
> If its easy to re-instate that, that would be great. Otherwise if we can check that the version shows up somewhere in the help, that would be sufficient.

If people are relying on parsing `--help` output, we will break them; there are no guarentees.

We intentionally dropped this to recover vertical space.  If you want precedence / consistency with other rust commands, `cargo` does not include the version.